time start_time

geometry_msgs/Point[] pos_pts
geometry_msgs/Vector3 init_v
geometry_msgs/Vector3 init_a

float64[] t_pts
